### PR TITLE
DDCE 960:Section 3 Warm-up Page: The charity's governing document

### PR DESCRIPTION
**Pull request by prameya-phatak**
# DDCE-960

##New feature

 Section 3 Warm-up Page: The charity's governing document

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/regulatorsAndDocuments/CharityGoverningDocumentView.scala.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    @p(Html(messages("charityGoverningDocument.p1")))
+
+    @bullets(
+    "charityGoverningDocument.b1",
+    "charityGoverningDocument.b2"
+    )
+
+    @p(Html(messages("charityGoverningDocument.p2")))
+
+    @link(controllers.regulatorsAndDocuments.routes.SelectGoverningDocumentController.onSubmit(NormalMode).url, "site.continue",Some("linkButton"))
```
Comment (abhijeetardale) [inline app/views/regulatorsAndDocuments/CharityGoverningDocumentView.scala.html:43]: please check the URL, might be onPageLoad

File: test/views/regulatorsAndDocuments/CharityGoverningDocumentViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        val view = viewFor[CharityGoverningDocumentView](Some(emptyUserAnswers))
+        view.apply()(fakeRequest, messages, frontendAppConfig)
+      }
+
+      behave like normalPage(applyView(), messageKeyPrefix, section = Some(messages("regulatorsSummary.section")))
+
+      behave like pageWithAdditionalGuidance(applyView(), messageKeyPrefix,
+        "p1", "b1", "b2","p2")
+
+      behave like pageWithHyperLink(applyView(), "linkButton", controllers.regulatorsAndDocuments.routes.SelectGoverningDocumentController.onSubmit(NormalMode).url, messages("site.continue"))
```
Comment (abhijeetardale) [inline test/views/regulatorsAndDocuments/CharityGoverningDocumentViewSpec.scala:41]: check the url please

File: app/controllers/regulatorsAndDocuments/CharityGoverningDocumentController.scala [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import config.FrontendAppConfig
+import controllers.LocalBaseController
+import controllers.actions._
+import javax.inject.Inject
+import play.api.mvc._
+import repositories.UserAnswerRepository
+import views.html.regulatorsAndDocuments.CharityGoverningDocumentView
+
+
+class CharityGoverningDocumentController @Inject()(
```
Comment (abhijeetardale) [inline app/controllers/regulatorsAndDocuments/CharityGoverningDocumentController.scala:28]: can you rename this as StartGoverningDocumentController and simillilar name for view and message prefix

Comment (prameya-phatak) [reply app/controllers/regulatorsAndDocuments/CharityGoverningDocumentController.scala]: updated

**Pull request merged**